doc/make.jl

```diff
@@ -0,0 +1,34 @@
+using Documenter, Clustering
+
+makedocs(
+    source = "source",
+    format = Documenter.HTML(prettyurls=false),
+    sitename = "Clustering.jl",
+    modules = [Clustering],
+    pages = [
+        "Clustering.jl Docs" => "index.md",
```

Comment:
"docs" is a bit casual, and repeating the name of the package isn't very useful. FWIW in DataFrames we just say "Introduction".